resqpy/property/property_collection.py

```diff
@@ -90,6 +89,7 @@ def __init__(self, support = None, property_set_root = None, realization = None)
         ]  # list of (uuid, file_name, keyword, cached_name, discrete, uom, time_index, null_value,
         #                                   min_value, max_value, property_kind, facet_type, facet, realization,
         #                                   indexable_element, count, local_property_kind_uuid, const_value, points)
+        self.guessing = False
```

Comment:
Should this be self.guess_warning = False ?

Reply:
Good spot, just updated this now